doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -232,6 +233,14 @@ Example of regular TLS logging:
       "issuerdn": "C=US, O=Google Inc, CN=Google Internet Authority G2"
   }
 
+Example of regular TLS logging for resumed sessions:
+
+::
+
+  "tls": {
+      "session": "resumed"
```

Comment:
I think a bool is better if you wanted to search on "resumed" sessions. Technically it doesn't matter, but its easier to think of it as a bool I think, if there are no other values that make sense.

Just to confirm, "sni" still exists so we can back track to the initial session?